src/modules/sdk/objections/axios.client.ts

```diff
@@ -4,15 +4,18 @@ import { ApiError } from "./types";
 
 export const STATUS_NO_RESPONSE = -1;
 export const HTTP_POST: Method = "post";
+export const HTTP_PATCH: Method = "patch";
 
 /**
  * An axios config with common elements for API calls.
  * @param {Method} httpMethod the http method to use eg. get, post
  * @param {string} url of the api endpoint
  * @param {string} bearerToken token for API call
  */
-export const getAxiosRequestConfig = (httpMethod: Method, url: string, bearerToken: string): AxiosRequestConfig => {
+export const getAxiosRequestConfig = (
+    httpMethod: Method, url: string, bearerToken: string, requestData): AxiosRequestConfig => {
```

Comment:
i would either make requestData optional with question mark - requestData? and in turn make data? optional or maybe rename getAxiosRequestConfig to getBaseAxiosRequestConfig (like we did for extensions) and then each function can add to it
eg https://github.com/companieshouse/extensions-web/blob/master/src/client/apiclient.ts#L116

Reply:
👍 Implementing the extensions approach.